src/behat/Container.php

```diff
@@ -240,24 +240,33 @@ public function getContainerIpAddress(string $service): string
     }
 
     /**
-     *  Get the container(s) logs.
+     * Get the container(s) logs.
      *
-     *  @param $service Service name.
-     *
-     *  @return string The logs as a string.
+     * @return string The logs as a string.
      */
-    public function getLogs($service = '')
+    public function getLogs()
     {
-        $cmd = 'docker-compose -f ' . $this->composeFile . ' -p ' . $this->id . ' logs -t --no-color';
-        if (!empty($service)) {
-            $cmd .= ' ' . $service;
-        }
+        $command = sprintf(
+            'docker-compose -f %s -p %s %s logs -t --no-color',
+                $this->composeFile,
+                $this->id,
```

Comment:
Therorically better for security.
```suggestion
                escapeshellarg($this->composeFile),
                escapeshellarg($this->id),
```